plugins/module_utils/network/sonic/config/lst/lst.py

```diff
@@ -0,0 +1,575 @@
+        :param have: the current configuration as a dictionary
+        :rtype: A list
+        :returns: the commands necessary to migrate the current configuration
+                  to the desired configuration
+        """
+        commands = []
+        requests = []
+        state = self._module.params['state']
+        diff = get_diff(want, have, TEST_KEYS)
+
+        if state == 'merged':
+            commands, requests = self._state_merged(diff)
+        elif state == 'replaced':
+            commands, requests = self._state_replaced(want, have, diff)
+        elif state == 'overridden':
+            commands, requests = self._state_overridden(want, have)
+        elif state == 'deleted':
+            commands, requests = self._state_deleted(want, have)
+        return commands, requests
+
+    def _state_merged(self, diff):
+        """
+        The command generator when state is merged
+        :rtype: A list
+        :returns: the commands necessary to merge the provided into
+                  the current configuration
+        """
+        commands = diff
+        requests = self.get_modify_lst_request(commands)
+
+        if commands and len(requests) > 0:
+            commands = update_states(commands, 'merged')
+        else:
+            commands = []
+
+        return commands, requests
+
+    def _state_replaced(self, want, have, diff):
+        """
+        The command generator when state is replaced
+        :rtype: A list
+        :returns: the commands necessary to migrate the current configuration
+                  to the desired configuration
+        """
+        commands = []
+        mod_commands = []
+        replaced_config, requests = self.get_replaced_config(want, have)
+
+        if replaced_config:
+            commands.extend(update_states(replaced_config, 'deleted'))
+            mod_commands = want
+        else:
+            mod_commands = diff
+
+        if mod_commands:
+            mod_request = self.get_modify_lst_request(mod_commands)
+
+            if mod_request:
+                requests.append(mod_request)
+                commands.extend(update_states(mod_commands, 'replaced'))
+
+        return commands, requests
+
+    def _state_overridden(self, want, have):
+        """
+        The command generator when state is overridden
+        :rtype: A list
+        :returns: the commands necessary to migrate the current configuration
+                  to the desired configuration
+        """
+        commands = []
+        requests = []
+        del_commands = get_diff(have, want, TEST_KEYS)
+        self.remove_default_entries(del_commands)
+
+        if del_commands:
+            is_delete_all = True
+            del_requests = self.get_delete_lst_requests(del_commands, have, is_delete_all)
+            requests.extend(del_requests)
+            commands.extend(update_states(have, 'deleted'))
+            have = {}
+
+        if not have and want:
```

Comment:
Please handle scenario when 'have' is subset of 'want'. (nothing to delete, but new config is to be added)